src/templates/base/2019/base_ebook.html

```diff
@@ -27,125 +32,101 @@
-                    </a>
-                    {% endif %}
-                  {% endif %}
-                  </div>
-              {% endfor %}
-            </div>
-          </section>
-        {% endfor %}
-        <section class="part">
-          <h2 class="part-name">{{ self.appendix() }}</h2>
-          <div class="chapters">
-            <div class="chapter">
-              <a href="#methodology">{{ self.methodology_title() }}</a>
-            </div>
-            <div class="chapter">
-              <a href="#contributors">{{ self.contributors_title() }}</a>
-            </div>
+          <div class="chapter">
+            <a href="#contributors">{{ self.contributors_title() }}</a>
           </div>
-        </section>
-      </section>
-
-      <section id="foreword">
-        <h1>{{ self.foreword_title() }}</h1>
-        {{ self.foreword() | replace('/%s/%s/contributors#' % (lang, year), '#contributors-') }}
-      </section>
-
-    {{ self.chapters() }}
-
-    <section class="chapter" id="methodology">
-      <div class="subtitle">Appendix A</div>
-      {{ get_ebook_methodology(lang,year) | safe }}
-    </section>
-
-    <section class="chapter" id="contributors">
-      <div class="subtitle">Appendix B</div>
-    
-        <h1 class="title title-lg">
-          {{ self.contributors_title() }}
-        </h1>
-        <div class="team">
-            <img src="/static/images/avatars/1.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/8.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/4.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/0.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/11.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/9.jpg" alt="" height="100" width="100" loading="lazy" />
         </div>
-        <p>{{ self.contributors_description() }}</p>
-        <ul class="contributors">
-        {% for id, contributor in config.contributors.items() %}
-        <li class="contributor" id="contributors-{{ id }}">
-          <img class="contributor-avatar" src="{{contributor.avatar_url}}" height="100" width="100" alt="{{ contributor.name }} avatar" />
-          <div>
+      </div>
+    </div>
+  </section>
+
+  <section id="foreword">
+    <h1>{{ self.foreword_title() }}</h1>
+    {{ self.foreword() | replace('/%s/%s/contributors#' % (lang, year), '#contributors-') }}
+  </section>
+
+  {{ self.chapters() }}
+
+  <section class="chapter" id="methodology">
+    <div class="subtitle">Appendix A</div>
+    {{ get_ebook_methodology(lang,year) | safe }}
+  </section>
+
+  <section class="chapter" id="contributors">
+    <div class="subtitle">Appendix B</div>
+  
+    <h1 class="title title-lg">
+      {{ self.contributors_title() }}
+    </h1>
+    <div class="team">
```

Comment:
Flexbox with `space-between` so these images spread out across the page width

Reply:
Again Flexbox not supported in WeasyPrint, but works in Prince! So fixed.
Please approve using this😀